packages/app-staking/src/Account/index.tsx

```diff
@@ -263,7 +265,7 @@ class Account extends React.PureComponent<Props, State> {
     return (
       <div className='staking--Account-detail'>
         <label className='staking--label'>{t('stash')}</label>
-        <AddressMini value={stashId} />
+        <AddressMini withBalance={false} withBonded value={stashId} />
```

Comment:
Here I believe just the bonded value is all-ok. (Since we are really just interested in that)